File: mobileos-cloud/mobileos-services/src/main/java/br/com/wsilva/mobileos/entity/VeiculoTipoEntity.java
package br.com.wsilva.mobileos.entity;

import lombok.Data;

import javax.persistence.*;

@Entity
@Table(name = "mos_veiculo_tipo")
@Data
public class VeiculoTipoEntity {
    @Id
    @Column(name = "vetp_id")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    @Column(name = "vetp_dsveiculotipo")
    private String descricao;

    public VeiculoTipoEntity() {
    }
}

File: mobileos-android/app/src/main/java/br/com/wsilva/mobileos/model/entity/VeiculoTipoEntity.kt
package br.com.wsilva.mobileos.model.entity

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "veiculo_tipo")
class VeiculoTipoEntity(var id: Long,
                        var descricao: String)
{
    @ColumnInfo(name = "_id")
    @PrimaryKey(autoGenerate = true)
    var uid: Long = 0
}